### PR TITLE
GH-16 tests on ImageService

### LINKED ISSUE #16: create ImageService (and write tests)

### PR DESCRIPTION
**Pull request by safgbad**

### CODE REVIEW

File: src/main/java/ru/skypro/flea/service/impl/ImageServiceImpl.java [hunk 1]
```diff
@@ -30,31 +32,28 @@ public ImageServiceImpl(String imagesPath) {
   public void checkCatalogue() {
     File catalogue = new File(imagesPath);
     if (!catalogue.exists()) {
-      boolean result = catalogue.mkdir();
+      boolean result = catalogue.mkdirs();
```
Comment (safgbad) [inline src/main/java/ru/skypro/flea/service/impl/ImageServiceImpl.java:35]: mkdir**s** to create parent catalogues also (mkdir doesn't do this)

File: src/main/java/ru/skypro/flea/service/impl/ImageServiceImpl.java [hunk 2]
```diff
@@ -30,31 +32,27 @@ public ImageServiceImpl(String imagesPath) {
...
 
   @Override
   public void saveImage(MultipartFile multipartFile, String fileName) {
     String extension = "";
-    String originalFileName = multipartFile.getOriginalFilename();
-    if (originalFileName == null) {
-      throw new RuntimeException();
-    }
-    int i = originalFileName.lastIndexOf(".");
+    String multipartFileName = multipartFile.getName();
```
Comment (safgbad) [inline src/main/java/ru/skypro/flea/service/impl/ImageServiceImpl.java:49]: as it turned out we do not need originalFileName
and name can not be null
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/multipart/MultipartFile.html#getName()

**Review by LoliPis — APPROVED** [review]
Good work, man! Thanks for your remarks!

File: src/main/java/ru/skypro/flea/service/impl/ImageServiceImpl.java (hunk 2 above)
Comment (safgbad) [reply src/main/java/ru/skypro/flea/service/impl/ImageServiceImpl.java:49]: nevermind...

**Pull request merged**